src/main/java/com/appleaster/exception/AppleasterException.java

```diff
@@ -0,0 +1,6 @@
+package com.appleaster.exception;
+public class AppleasterException extends Exception {
```

Comment:
Perhaps extending from RuntimeException might be more appropriate as AppleasterException is thrown during run time as opposed to during compile time.